project/src/components/Header/header.jsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { AppRoute, AuthorizationStatus } from '../../const';
 import { logout } from '../../store/api-action';
+import { getAuthorizationStatus, getEmail } from '../../store/user/selectors';
+
+function Header() {
+  const authorizationStatus = useSelector(getAuthorizationStatus);
+  const email = useSelector(getEmail);
+
+  const dispatch = useDispatch();
+  const logoutApp = () => {
+    dispatch(logout());
+  };
```

Comment:
```suggestion
  const logoutApp = () => dispatch(logout());
```

можно же в одну строку написать)